JASP-Engine/JASP/R/bayesianmetaanalysis.R

```diff
@@ -386,7 +386,7 @@ BayesianMetaAnalysis <- function(jaspResults, dataset, options) {
   }
   
   if(isTryError(p)){
-    .quitAnalysis(gettextf("The model could not fit. Error while running R code from the metaBMA package: %s", .extractErrorMessage(p))) 
```

Comment:
Does this error message ever provide some useful information? Because "Try different settings" is a very general suggestion

Reply:
So far, I have only seen the error messages "no parameter specified (pars is empty)" and "not enough unique values". It seems to give the first one when the analysis does not work (i.e. the model doesn't fit) and the second one appeared with the Mac/callr issue (no idea why). 

The model does not fit, for example, when the data is mostly negative/positive while the prior only contains positive/negative values. I believe there were other situations as well, but I cannot think of them right now.

I don't really know how to deal with this elegantly. If you have any suggestions, I am happy to hear them!

Reply:
Yeah those error messages aren't great. But maybe you could expand a tiny bit upon "try different settings"; do you mean certain checkboxes? Interval sizes? Variables?

Reply:
@sophieberkhout Could you answer/implement Tim's suggestion today, than we can push your PR, and start a second testing round

Reply:
How is this?

Reply:
Yes that's a much more informative error message :)